src/test/java/org/gaul/s3proxy/S3AwsSdkTest.java

```diff
@@ -135,8 +136,9 @@ public void tearDown() throws Exception {
         }
     }
 
+
     @Test
-    public void testAwsV2Signature() throws Exception {
+    public void testAwsV2SignatureWithOverrideParameters() throws Exception {
```

Comment:
Can you break this into its own test?  I want to retain the existing `testAwsV2Signature`.